views/onboarding.ejs

```diff
@@ -0,0 +1,111 @@
+    <div class="notification"><%- messages %></div>
+    <% } %>
+
+    <div class="panel margin-top-m">
+
+        <div class="beta-banner"></div>
+
+        <h3>Créer ma fiche Github</h3>
+
+        <p>Bienvenue dans la communauté</p>
+        <p>Crée ta fiche Github pour avoir accès aux différents outils de la communauté</p>
+
+        <form action="/onboarding" method="POST">
+            <h4>Tes infos persos</h4>
+
+            <div class="form__group">
+                <label>
+                    Ton nom sera affiché sur la page <a href="https://beta.gouv.fr/communaute/"
+                        target="_blank">Communauté</a> du site BetaGouv.
+                </label>
+                <label for="firstName">
+                    <strong>Prénom (obligatoire)</strong>
+                </label>
+                <input name="firstName" required>
+                <label for="lastName">
+                    <strong>Nom de famille (obligatoire)</strong>
+                </label>
+                <input name="lastName" required>
+            </div>
+
+            <div class="form__group">
+                <label for="website">
+                    <span>Ton site personnel</span><br />
+                    Il sera affiché sur la page <a href="https://beta.gouv.fr/communaute/"
+                        target="_blank">Communauté</a> du site BetaGouv.
+                </label>
+                <input name="website">
+            </div>
+
+            <div class="form__group">
+                <label for="github">
+                    <span>Utilisateur Github</span><br />
+                    Si tu as un compte, tu peux être ajouté à l'organisation Github BetaGouv
+                </label>
+                <input name="github">
+            </div>
+
+
+            <h4>Ta mission</h4>
+            <div class="form__group">
+                <label for="role">
+                    <strong>Ton rôle chez BetaGouv (obligatoire)</strong><br />
+                    Développeuse, Intrapreneur, Chargée de déploiement, Coach, UX Designer...
+                </label>
+                <input name="role" required>
+            </div>
+
+            <div class="form__group">
+                <label for="start">
+                    <strong>Début de ta mission (obligatoire)</strong><br />
+                    En format YYY-MM-DD, par exemple : 2020-01-31
+                </label>
+                <input pattern="^\d{4}-\d{2}-\d{2}$" name="start" required>
+            </div>
+
+            <div class="form__group">
+                <label for="end">
+                    <strong>Fin de ta mission (obligatoire)</strong><br />
+                    En format YYY-MM-DD, par exemple : 2020-01-31
+                </label>
+                <input pattern="^\d{4}-\d{2}-\d{2}$" name="end" required>
+            </div>
+
+            <div class="form__group">
+                <label for="status">
+                    <strong>Ton statut (obligatoire)</strong><br />
```

Comment:
je mettrai dans l'ordre statut puis employer, qui apparaissent in fine ensemble sur la fiche d'un membre. et du coup je mettrai ma startup en dessous

Reply:
Pourquoi pas, il ne doit pas en avoir beaucoup non ?